### ISSUE TITLE
Domain (WHOIS) expiry check

### ISSUE THREAD

**Issue by martyn-meister** [louislam/uptime-kuma#3454]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

New Monitor

### 🔖 Feature description

It would be super nice to be sure we don't miss our domain registrar's expiry, so it would be really nice if like the SSL expiry check we could on a much less frequent basis do a whois check to ensure no lost domains.

### ✔️ Solution

Basically an additional part of any check that could be enabled on say the first check of the day/week to do a whois lookup against the domain and see if it is due to expire soon.  This could then be notified like the TLS expiry notification.

### ❓ Alternatives

At the moment this would have to be done using a separate tool.

### 📝 Additional Context

This is obviously not a high priority request, just would be cool to include.

**Comment by CommanderStorm** [comment]
I think this is a duplicate of https://github.com/louislam/uptime-kuma/issues/944
If you agree, could you please close this Issue, as duplicates only create immortal zombies and are really hard to issue-manage?
If not, what makes this issue unique enough to require an additional issue? (Could this be integrated into the issue linked above?) ^^

**Comment by martyn-meister** [comment]
Agree, sorry, I did have a brief look but the key words are in almost every issue is my excuse!

**Comment by CommanderStorm** [comment]
> I did have a brief look but the key words are in almost every issue is my excuse!

For the next time, try searching for likely correlated words like ["WHOIS"](https://github.com/louislam/uptime-kuma/issues?q=is%3Aissue+WHOIS+) or ["Domain expiry"](https://github.com/louislam/uptime-kuma/issues?q=is%3Aissue+Domain+expiry+)
